linkerd/config_templates/emojivoto.yml

```diff
@@ -0,0 +1,197 @@
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: emoji
+  namespace: {{.namespace}}
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: voting
+  namespace: {{.namespace}}
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: web
+  namespace: {{.namespace}}
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  name: emoji
+  namespace: {{.namespace}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: emoji-svc
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: emoji-svc
+    spec:
+      serviceAccountName: emoji
+      containers:
+      - env:
+        - name: GRPC_PORT
+          value: "8080"
+        image: buoyantio/emojivoto-emoji-svc:v7
```

Comment:
I wonder if we will want to unpin the tagged version at some future point. ":v7"

Reply:
Literally just released v8 yesterday :)

Reply:
```
Using default tag: latest
Error response from daemon: manifest for buoyantio/emojivoto-web:latest not found
```
I would have loved to just use it from the URL directly, which is what I tried first. The issue was the hardcoded namespace.
Please let me know if you guys know of an alternative url where the namespace is not hardcoded.

Reply:
actually, never mind, I was able to force the namespace from the code. . . this is now removed

Reply:
That's too funny. :) Thanks for pointing this out, @wmorgan.